features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
     | type                | RaadpleegMetBurgerservicenummer                                                         |
-    | burgerservicenummer | 555550003                                                                               |
+    | burgerservicenummer | 000007895                                                                               |
     | fields              | ouders.ouderAanduiding,ouders.burgerservicenummer,ouders.geslacht,ouders.naam.voornamen |
-    Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
     | naam                  | waarde    |
+	| burgerservicenummer   | 000007896 |
+    | geslacht.code         | V         |
+    | geslacht.omschrijving | vrouw     |
+	| naam.voornamen        | Ellen     |
     | ouderAanduiding       | 1         |
-    | burgerservicenummer   | 555550004 |
+    En heeft de persoon een 'ouder' met de volgende gegevens
+    | naam                  | waarde    |
+	| burgerservicenummer   | 000007897 |
     | geslacht.code         | M         |
     | geslacht.omschrijving | man       |
-    En heeft de ouder met ouderAanduiding '1' de volgende 'naam' gegevens
-    | naam      | waarde |
-    | voornamen | Karel  |
+	| naam.voornamen        | Karel     |
+    | ouderAanduiding       | 2         |
 
 @gba
 Rule: Wanneer alleen gegevens in groep 81, 82, 83, 84, 85 en/of 86 zijn opgenomen en geen gegevens in groep 1, 2, 3, 4 of 62, dan wordt de ouder niet opgenomen
   Gebruik van de fields parameter heeft geen invloed op het leveren van een ouder
 
   @gba
   Scenario: vondeling (onbekende ouder met ouderaanduiding "1" en geen ouder met ouderaanduiding "2")
-    Gegeven de persoon met burgerservicenummer 999993008 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Geslachtsnaam | Geboortedatum (03.10) | Geslachtsaanduiding (04.10) | Datum ingang familierechtelijke betrekking (62.10) |
-    | 2         |                    | .             |                       | V                           | 20190614                                           |
-    | 3         |                    |               |                       |                             |                                                    |
-    En in categorie 3 komt alleen groep 81 en groep 85 voor
+    Gegeven de persoon met burgerservicenummer '000007896' heeft een ouder '1' met de volgende gegevens
+    | datum ingang familierechtelijke betrekking (62.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+    | 20190614                                           | .                     | V                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+	| aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
+	| 2•E0001            | 20160518                        |
```

Comment:
is er iets fout gegaan in aktenummer? hier staat een "•" (bullit) in. beetje vreemd

Reply:
Vind ik ook maar het aktenummer bestaat uit 7 posities waarvan de eerste 3 conform Tabel 39 (Akteaanduiding) dienen te zijn. Uit bijgaand voorbeeld van die tabel:

![image](https://user-images.githubusercontent.com/20185784/191673815-e93b4069-dc21-4f01-9cc3-a2ee99f0c432.png)

zie je dat die bullet er echt in zit.